### DISCUSSION TITLE
유니티에서 ZEPETO 앱 테스트하려고 QR코드 연결하면 자꾸 접속에 실패한다고 뜹니다.

### DISCUSSION THREAD

**Discussion by srlee010** [naverz/zepeto-studio-kor#226] [5. Q&A]
안녕하세요 유니티를 통해 제페토 월드맵을 만들기 시작한 초보입니다.
제페토가이드와 유니티 learn을 통해 열심히 공부하고 있는데요.. 

유니티에서 ZEPETO 앱 테스트하려고 QR코드 찍으면  자꾸 접속에 실패한다고 뜹니다.
다시 시도해달라고 해서 간격두고 계속 접속중인데 오류가 뜨는데 해결해주실 수 있나요?
일반 사용자들이 월드 만들기 너무 어렵네요 ㅠㅠㅠ

**Comment by mayko1234** [comment]
혹시 운영체제, 윈도우즈 사용하시나요?
방화벽을 해제하셨는지 확인해보세요. 
첨부한 사진처럼 방화벽을 턴오브하셔야 합니다. 
![image](https://user-images.githubusercontent.com/97988822/150019407-aaf2e03d-60e2-46e6-b6dd-80c9cc2a972b.png)

**Reply by Zimins** [reply]
저도 같은 문제가 있어서 가이드 주신대로 보이는 방화벽 설정은 다 해봤는데 연결이 잘안되는것으로 보이네요.

* 재부팅이 필요할까요?
* 방화벽을 열어줘야 하면, 혹시 공유기에서 포트 포워딩 설정은 필요없나요?

<img width="901" alt="화면 캡처 2022-01-19 110920" src="https://user-images.githubusercontent.com/3432994/150050364-edc59271-78eb-47a5-94af-4ce59d47b6bd.png">
<img width="809" alt="화면 캡처 2022-01-19 110839" src="https://user-images.githubusercontent.com/3432994/150050370-dac6b4b7-7deb-4f55-b536-d8acc807a54d.png">
<img width="785" alt="화면 캡처 2022-01-19 110822" src="https://user-images.githubusercontent.com/3432994/150050381-6875fe08-3fa2-43be-910d-f633ded888b4.png">

**Comment by strangestraw** [comment]
PC의 IP 세팅 확인해보세요.
아래 화면에서 빨간 부분 누르면 IP 설정이 가능합니다.
![image](https://user-images.githubusercontent.com/5335151/150051420-0602934a-1f36-4041-a45a-a5106e1e8628.png)

**Reply by Zimins** [reply]
오 확인하지 못한 부분이네요.
실제로 보니 제 경우는 docker 설정때문인지 잘못된 ip 로 되어있던것으로 보입니다.
다만 그 값을 바꿔도 여전히 연결은 되지 않고 있습니다. 

<img width="496" alt="화면 캡처 2022-01-19 130543" src="https://user-images.githubusercontent.com/3432994/150062067-c2618ecd-4826-4ca3-9a1a-dfa48d976295.png">

혹시 유니티 실행 환경은 유선랜, 모바일 기기는 같은 공유기에 무선랜으로 들어가 있는데 이것도 맞추는게 필요할까요?

**Reply by strangestraw** [reply]
같은 망에 있는 경우 문제가 없고, 방화벽까지 다 꺼놓으셨다면 되는게 정상입니다.

**Reply by Zimins** [reply]
감사합니다 다시 확인해볼게요~

**Reply by Zimins** [reply]
재부팅을 몇번 해보니 링크에 접속 됩니다.
다만 현재 화면이 아닌 이전에 빌드 했던 화면이 바뀌지 않고 보이는 현상이 있어서 나중에 문의 드리겠습니다. 감사합니다.

**Reply by strangestraw** [reply]
작업하시는 씬 **저장**하시고 Rebuild 후 접속해보세요~

**Reply by srlee010** [reply]
도움주신분들 모두 답변 감사합니다
